packages/vite/src/node/plugins/json.ts

```diff
@@ -43,6 +44,9 @@ export function jsonPlugin(
       if (!jsonExtRE.test(id)) return null
       if (SPECIAL_QUERY_RE.test(id)) return null
 
+      // strip UTF-8 BOM
```

Comment:
```suggestion
```

Reply:
Yes, in fact I'm thinking about removing these comments, haha